aws/data_aws_unity_catalog_policy.go

```diff
@@ -92,6 +93,12 @@ func validateSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Required: true,
 		},
+		"aws_partition": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.StringInSlice([]string{"aws", "aws-cn", "aws-us-gov"}, false),
```

Comment:
can we define this slice as constant or var, so it will be accessible to all data sources?

Reply:
Good call! Moved as a const to common